### PR TITLE
tikv: add server.end-point-slow-log-threshold config (#4327)

### PR DESCRIPTION
**Pull request by ti-srebot**
cherry-pick #4327 to release-3.1

---

Signed-off-by: Andy Lok <andylokandy@hotmail.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

`server.end-point-slow-log-threshold` configuration is added.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

https://github.com/tikv/tikv/pull/9061

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/5013
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -18,6 +18,50 @@ This document only describes the parameters that are not included in command-lin
 + Default value: `1`
 + Minimum value: `1`
 
+<<<<<<< HEAD
```
Comment (yikeke) [inline tikv-configuration-file.md:21]: @breeswish hi please help confirm if the changes apply to 3.0 version, thanks~

Comment (breeswish) [reply tikv-configuration-file.md:21]: Hi, this change is included in 3.0 version. Thanks!

Comment (yikeke) [reply tikv-configuration-file.md:21]: Hi @breeswish @andylokandy Could you please help resolve the conflicts then? I'm not sure about the final version that is technically right.

And "However the doc should be updated after corresponding new version is release" means this PR should not be merged until the new 3.0 version is released?

**Review by yikeke — APPROVED** [review]
LGTM

**Pull request merged**